examples/jsm/nodes/accessors/ColorsNode.d.ts

```diff
@@ -0,0 +1,11 @@
+import { NodeBuilder } from '../core/NodeBuilder';
+import { TempNode } from '../core/TempNode.js';
+
+export class ColorsNode extends TempNode {
```

Comment:
`ColorsNode` has no `nodeType` defined. Is this intended?

Reply:
> The class ColorsNode has no nodeType defined. Is this intended?

No, no! I must have forgotten to add 👍

Reply:
Okay, we can resolve that later 😉